### PR TITLE
Resolve eslint errors

### PR DESCRIPTION
**Pull request by karatsuba**
hi @perry-mitchell!
I saw a lot of eslint errors after my first PR, so I decided to fix them.

### CODE REVIEW

File: source/setup/js/UnlockArchive.jsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import { hashHistory } from "react-router";
+import { noop } from "lodash";
```
Comment (perry-mitchell) [inline source/setup/js/UnlockArchive.jsx]: There's no need to import lodash here. `NOOP` (playfully renamed to `NOPE` here) is classic nomenclature for "no operation", and is commonly and empty function. We don't need a library to create an empty function 😅.

**Comment by perry-mitchell** [comment]
Just one change thanks @karatsuba  - Good work.

File: source/setup/js/UnlockArchive.jsx (hunk 1 above)
Comment (karatsuba) [reply source/setup/js/UnlockArchive.jsx]: I just saw `lodash` in `node_modules` and decided to use it :smile:.
And yes, I agree, there is no need to do that.

btw, travis-ci is still failing on this step - `webpack --progress` + I don't have errors locally.

**Comment by karatsuba** [comment]
@perry-mitchell, I pushed my changes.

**Pull request merged**